static/js/manipulation.js

```diff
@@ -478,10 +478,8 @@
 	        }
 
 	    var fkeys = Object.keys(_filterQueue);
-	    var flen = fkeys.length;
-	    var flenScale = 1 / flen;
 
-	    fkeys.map(function (filt) {});
+	    fkeys.map(function () {});
```

Comment:
This again, same as in here:
https://github.com/DDMAL/pixel_wrapper/blob/6b769fb917430992add15258e43445c65865e10a/source/js/plugins/_filters.js#L19-L21

Reply:
This is the compiled / webpacked file so it's actually the same line as in `_filters.js`